libelfloader/elf_ld.c

```diff
@@ -47,8 +48,9 @@ BOOLEAN image_copy(void *dest, module_file_info_t *file_info,
 		 (file_info->runtime_addr + file_info->runtime_image_size))) {
 		return FALSE;
 	}
-	memcpy(dest, src, bytes_to_copy);
-	return TRUE;
+
+	ret = memcpy_s(dest, bytes_to_copy, src, bytes_to_copy);
```

Comment:
I think you don't need to do change here. buf_len is checked in line 46~49. suggest to disposition it through mail.

Reply:
If and when we change from memcpy to memcpy_s we should do it everywhere and do not leave old code. Part of code cleanup forced by Klocwork.